client/src/views/AdminsPage.tsx

```diff
@@ -0,0 +1,43 @@
+import React, {useState} from 'react';
+import DoctorTable, {Doctor} from '../components/DoctorTable';
+import { useNavigate } from 'react-router-dom'; // Import useNavigate
+
+
+const AdminPage: React.FC = () => {
+    const navigate = useNavigate(); // Initialize useNavigate
+    const [doctorData, setDoctorData] = useState<Doctor[]>([
+      { _id: '111', firstName: 'John', lastName: 'Doe', SSN: '987654', username: 'jdoe', password: 'password123' },
+      { _id: '222', firstName: 'Jane', lastName: 'Smith', SSN: '456789', username: 'jsmith', password: 'password456' }
+    ]);
```

Comment:
Has to be altered after integration to use json array, not hardcoded data.

Reply:
Absolutely, once we integrate with the endpoints for doctors and patients, we can fetch real data and replace the dummy data placeholders. This should be straightforward to implement.